crates/client/rpc-core/src/lib.rs

```diff
@@ -27,6 +28,21 @@ use starknet_core::types::{
 #[derive(Serialize, Deserialize)]
 pub struct Felt(#[serde_as(as = "UfeHex")] pub FieldElement);
 
+#[derive(Serialize, Deserialize)]
+pub struct PredeployedAccountWithBalance {
+    pub account: PredeployedAccount,
+    pub balance: FieldElement,
+}
+
+pub type PredeployedAccountsList = Vec<PredeployedAccountWithBalance>;
```

Comment:
Not necessary. Require the lector to do one more jump in the code to understand. Remove it.